### PR TITLE
Update website to show details of the first Release

### PR DESCRIPTION
**Pull request by DLotts**
These are changes to the markdown mostly to the download.md, and new post with more details about the release.  More text about the what changed is welcome.  I just linked to the Jira generated list of changes.  Technically since this is the first release every line of code is a new change.

The HTML generated by Jekyll is found in asf-site and I will do a corresponding PR for that in a minute.
david.

### CODE REVIEW

File: content/_posts/2016-10-28-release-3.2.10.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+date: "2016-10-28"
+title: First Release
+description: details of the first release of Rya (Incubating)
+version: 3.2.10
+fullVersion: rya-incubating-3.2.10
+filename: rya-project-3.2.10-incubating-source-release
+githubRelease: https://github.com/apache/incubator-rya/releases/tag/rel%2Frya-incubating-3.2.10
```
Comment (adinancr) [inline content/_posts/2016-10-28-release-3.2.10.md]: Is a github release link necessary? I think we should only link to the Apache release. I would remove the githubRelease link.

File: content/_posts/2016-10-28-release-3.2.10.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+apacheRelease: https://dist.apache.org/repos/dist/release/incubator/rya/rya-incubating-3.2.10/
```
Comment (adinancr) [inline content/_posts/2016-10-28-release-3.2.10.md]: The download link for Apache releases is:
http://www.apache.org/dyn/closer.lua/incubator/rya/rya-incubating-3.2.10/rya-project-3.2.10-incubating-source-release.zip

File: content/_posts/2016-10-28-release-3.2.10.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+releaseNotes: https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12334209&styleName=Html&projectId=12319020
+categories: [release]
+tag: v3.2.10
+sha: 66d8b7f
```
Comment (adinancr) [inline content/_posts/2016-10-28-release-3.2.10.md]: Again, not sure the github release tag needs to show up on the website

File: content/_posts/2016-10-28-release-3.2.10.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+---
+{% comment %}
+Some of the variables above are used on the summary page that iterates through posts where catagories=release.
+{% endcomment %}
+
+This is Rya's first release, version {{ page.version }}.  It is available as a source code download:
+
+- [Source download from Apache]({{ page.apacheRelease }})
+- [Source download from Github]({{ page.githubRelease }})
+- [Release Tag in Github](https://github.com/apache/incubator-rya/commit/{{ page.sha }})
```
Comment (adinancr) [inline content/_posts/2016-10-28-release-3.2.10.md]: Do not direct users to a github download

File: content/_posts/2016-10-28-release-3.2.10.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+download has completed OK.
+
+For fast downloads, current source distributions are hosted on mirror servers;
+older source distributions are in the
+[archive](http://archive.apache.org/dist/incubator/{{ site.data.project.unix_name }}/).
+If a download from a mirror fails, retry, and the second download will likely
+succeed.
+
+For security, hash and signature files are always hosted at
+[Apache](https://www.apache.org/dist).
```
Comment (adinancr) [inline content/_posts/2016-10-28-release-3.2.10.md]: Use http://www.apache.org/dist/{{ site.data.project.incubator_slash_name }}/ as URL, not the generic Apache URL, as it is hard to navigate to the relevant Rya directory

File: content/download.md [hunk 6]
```diff
@@ -26,64 +26,67 @@ limitations under the License.
 
 ## {{ site.data.project.name }} Downloads
 
-There are no official Apache releases for {{ site.data.project.short_name }} yet.
-
-First Apache Incubator release coming soon!!
+The Official Apache release for {{ site.data.project.short_name }}.
```
Comment (adinancr) [inline content/download.md]: I would just delete this line ("The Official Apache release for Rya") as the website is already for Rya downloads)

File: content/download.md [hunk 7]
```diff
@@ -26,64 +26,67 @@ limitations under the License.
...
-
-TODO: uncoment this when we have official releases of Rya
-
 {{ site.data.project.name }} is released as a source artifact, and also through Maven.
 
 ### Source releases
 
-Release          | Date       | Commit   | Download
-:--------------- | :--------- | :------- | :-------
+Release          | Date       | Commit   | Download | Notes
```
Comment (adinancr) [inline content/download.md]: I would suggest removing the Commit column

File: content/download.md [hunk 8]
```diff
@@ -26,64 +26,67 @@ limitations under the License.
...
-First Apache Incubator release coming soon!!
+The Official Apache release for {{ site.data.project.short_name }}.
 
 If you are interested in helping out, please join our developers list [{{site.data.project.dev_list}}](mailto:{{site.data.project.dev_list_subscribe}}?subject=subscribe to Rya dev list) by sending an email to [{{site.data.project.dev_list_subscribe}}](mailto:{{site.data.project.dev_list_subscribe}}?subject=subscribe to Rya dev list).
 
-{% comment %}
-
-TODO: uncoment this when we have official releases of Rya
-
 {{ site.data.project.name }} is released as a source artifact, and also through Maven.
```
Comment (adinancr) [inline content/download.md]: Add the link to Maven Rya release here

File: content/download.md [hunk 9]
```diff
@@ -26,64 +26,67 @@ limitations under the License.
...
 
 ### Source releases
 
-Release          | Date       | Commit   | Download
-:--------------- | :--------- | :------- | :-------
+Release          | Date       | Commit   | Download | Notes
+:--------------- | :--------- | :------- | :------- | :----
 {% for post in site.categories.release %}{% comment %}
```
Comment (adinancr) [inline content/download.md]: Can you remove all the unnecessary endcomment/comment from the source? It is hard to read the code as it now.

File: content/download.md [hunk 10]
```diff
@@ -26,64 +26,67 @@ limitations under the License.
...
 {% endcomment %}{% endif %}{% comment %}
-{% endcomment %}<a href="{{ site.baseurl }}/history.html#{{ post.tag }}">{{ post.version }}</a>{% comment %}
+{% endcomment %} <a href="{{ post.url }}">{{ post.version }}</a>{% comment %}
 {% endcomment %} | {{ post.date | date_to_string }}{% comment %}
-{% endcomment %} | <a href="https://github.com/apache/{{ site.data.project.unix_name }}/commit/{{ post.sha }}">{{ post.sha }}</a>{% comment %}
-{% endcomment %} | <a href="{{ p }}/{{ v }}-src.tar.gz{{ q }}">tar</a>{% comment %}
+{% endcomment %} | <a href="{{ site.data.project.source_repository_mirror }}/commit/{{ post.sha }}">{{ post.sha }}</a>{% comment %}
+// Currently the mirrors have no signitures, so use dist.apache.org:
+{% endcomment %} | {% if post.filename %} <a href="{{ p }}/{{ post.filename }}{{ q }}">download</a>{% comment %}  
+{% endcomment %} , [signatures](https://dist.apache.org/repos/dist/release/{{ site.data.project.incubator_slash_name }}/{{ v }}){% comment %}
```
Comment (adinancr) [inline content/download.md]: Replace "signature" link with 3 separate direct links to .asc .md5 and .sha1 files

File: content/download.md [hunk 11]
```diff
@@ -26,64 +26,67 @@ limitations under the License.
...
+:--------------- | :--------- | :------- | :------- | :----
 {% for post in site.categories.release %}{% comment %}
 {% endcomment %}{% if post.fullVersion %}{% comment %}
 {% endcomment %}{% assign v = post.fullVersion %}{% comment %}
 {% endcomment %}{% else %}{% comment %}
 {% endcomment %}{% capture v %}apache-{{ site.data.project.unix_name }}-{{ post.version }}{% endcapture %}{% comment %}
 {% endcomment %}{% endif %}{% comment %}
 {% endcomment %}{% if forloop.index0 < 2 %}{% comment %}
-{% endcomment %}{% capture p %}http://www.apache.org/dyn/closer.lua?filename={{ site.data.project.unix_name }}/{{ v }}{% endcapture %}{% comment %}
+{% endcomment %}{% capture p %}http://www.apache.org/dyn/closer.lua?filename={{ site.data.project.incubator_slash_name }}/{{ v }}{% endcapture %}{% comment %}
```
Comment (adinancr) [inline content/download.md]: The link (http://www.apache.org/dyn/closer.lua?filename=incubator/rya/rya-incubating-3.2.10/rya-project-3.2.10-incubating-source-release&action=download) did not work well for me. The generated URL should be http://www.apache.org/dyn/closer.lua/incubator/rya/rya-incubating-3.2.10/rya-project-3.2.10-incubating-source-release.zip

File: content/download.md [hunk 12]
```diff
@@ -26,64 +26,67 @@ limitations under the License.
...
+
+<!-- older source distributions are in the
 [archive](http://archive.apache.org/dist/incubator/{{ site.data.project.unix_name }}/).
+--> 
 If a download from a mirror fails, retry, and the second download will likely
 succeed.
 
+
 For security, hash and signature files are always hosted at
 [Apache](https://www.apache.org/dist).
```
Comment (adinancr) [inline content/download.md]: Use http://www.apache.org/dist/{{ site.data.project.incubator_slash_name }}/ as URL, not the generic Apache URL, as it is hard to navigate to the relevant Rya directory

**Review by adinancr — COMMENTED** [review]
Thanks David for the PR. I have some comments, please see

File: content/download.md (hunk 9 above)
Comment (DLotts) [reply content/download.md]: I agree that it makes it hard to read, but the comments allow the table rows to wrap in the source but not in the markdown.  As you probably know, markdown requires the row to be on one line with vertical bars separating the columns.  The alternative is to reformat the script to generate HTML embedded in markdown.

File: content/_posts/2016-10-28-release-3.2.10.md (hunk 2 above)
Comment (DLotts) [reply content/_posts/2016-10-28-release-3.2.10.md]: I had used a similar link on the download page, originally included with the template.  They seem to the same except your link asks before redirecting to the mirror, where the template just redirects.  I'll change to yours in both download page and the release post.

File: content/_posts/2016-10-28-release-3.2.10.md (hunk 5 above)
Comment (DLotts) [reply content/_posts/2016-10-28-release-3.2.10.md]: done.

File: content/download.md (hunk 11 above)
Comment (DLotts) [reply content/download.md]: done.

File: content/download.md (hunk 12 above)
Comment (DLotts) [reply content/download.md]: done.

File: content/download.md (hunk 8 above)
Comment (DLotts) [reply content/download.md]: done.

File: content/_posts/2016-10-28-release-3.2.10.md (hunk 1 above)
Comment (DLotts) [reply content/_posts/2016-10-28-release-3.2.10.md]: done.

File: content/_posts/2016-10-28-release-3.2.10.md (hunk 4 above)
Comment (DLotts) [reply content/_posts/2016-10-28-release-3.2.10.md]: done.

File: content/download.md (hunk 6 above)
Comment (DLotts) [reply content/download.md]: done.

File: content/download.md (hunk 10 above)
Comment (DLotts) [reply content/download.md]: done.

**Pull request closed without merging**